process/process.go

```diff
@@ -53,9 +58,10 @@ func init() {
 }
 
 // New returns an object with Process interface accessing it
-func New(pid libpf.PID) Process {
+func New(pid, tid libpf.PID) Process {
```

Comment:
I didn't switch `Process` to accept `libpf.PIDTID` as the latter is only used with PID events, and I'd rather not couple it here too.